pkg/apis/db/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -110,6 +110,20 @@ func (in *CassandraClusterSpec) DeepCopyInto(out *CassandraClusterSpec) {
 		*out = new(ServicePolicy)
 		(*in).DeepCopyInto(*out)
 	}
+	if in.StorageConfigs != nil {
+		in, out := &in.StorageConfigs, &out.StorageConfigs
+		*out = make([]StorageConfig, len(*in))
+		for i := range *in {
+			(*in)[i].DeepCopyInto(&(*out)[i])
+		}
+	}
+	if in.SidecarConfigs != nil {
+		in, out := &in.SidecarConfigs, &out.SidecarConfigs
+		*out = make([]v1.Container, len(*in))
+		for i := range *in {
```

Comment:
I'd use something else than `i` like `index` or `idx`

Reply:
wait is it generated code ? if yes, nvm

Reply:
Yes this is generated ^^

Reply:
Yes this is generated ^^